### ISSUE TITLE
[Bug]: T8420 fails to start livestream after latest Eufy firmware (1.0.0.40)

### ISSUE THREAD

**Issue by smitty078** [homebridge-eufy-security/plugin#223]
### What happened?

streaming is broken on T8420 after Eufy firmware update. Issue is in security client. See bug report there: https://github.com/bropat/eufy-security-client/issues/251

### Device Type

Floodlight (Supported)

### Plugin Version

beta (Best Effort)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v14 (Supported)

### Operating System

Mac OS X 11+ (Supported)

### Relevant log output

```shell
see https://github.com/bropat/eufy-security-client/issues/251
```

### config

```shell
n/a
```

**Comment by thieren** [comment]
Hi @smitty078 

Thx for investigating and providing a fix for the lib. As soon as this is merged in the master I'll update the plugin with the new version.

**Comment by thieren** [comment]
This should be fixed with the latest `2.3.0` of the eufy-security-client, and `2.2.0-beta.10` of the plugin respectively.

Please check.

**Comment by smitty078** [comment]
Fixed for me
